### PR TITLE
Adds Highlight, Questions, and Teams

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -5,5 +5,7 @@
 What highlights can you come up with to share with the class?
 
 What do you really like? What can you find for fun facts related to any of these topics online?
+- The most fun part for Erick is using npm packages!
+- API integrations such as Auth0 and google apps.
```
Comment (Emeralds99) [inline HIGHLIGHTS.md:9]: Is Auth0 a security feature?

**Review by Emeralds99 — APPROVED** [review]
LGTM

File: HIGHLIGHTS.md (hunk 1 above)
Comment (hollisjamison) [reply HIGHLIGHTS.md:9]: Auth0 is a way to let another service be your authentication gateway.

Think about Doordash, how you can login via Google and Facebook. They use Auth0 authentication to authorize your login without Doordash ever getting your Facebook password.

File: QUESTIONS.md [hunk 2]
```diff
@@ -2,4 +2,7 @@
 
 What questions can you come up with?
 
+-  Can you walk us through the app deployment via heroku?
```
Comment (SBucharelli) [inline QUESTIONS.md:12]: I'd be interested in this too. Hollis spoke of it, but would like to know more.

**Review by SBucharelli — APPROVED** [review]

**Pull request merged**